cmd/viam-agent/main.go

```diff
@@ -183,7 +183,7 @@ func main() {
 	}
 
 	// wait until now when we (potentially) have a network logger to record this
-	globalLogger.Infof("Viam Agent Version: %s\nGit Revision: %s\n", agent.GetVersion(), agent.GetRevision())
+	globalLogger.Infof("Viam Agent Version: %s Git Revision: %s", agent.GetVersion(), agent.GetRevision())
```

Comment:
drive-by... didn't realize I'd copy/pasted the newlines from the CLI "version" output here (where it makes no sense.)